presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -22,7 +22,9 @@ Binary (WKB) form of spatial objects:
 * ``GEOMETRYCOLLECTION (POINT(2 3), LINESTRING (2 3, 3 4))``
 
 Use :func:`ST_GeometryFromText` and :func:`ST_GeomFromBinary` functions to
-create geometry objects from WKT or WKB.
+create geometry objects from WKT or WKB.  In WKT/WKB, the coordinate order is
```

Comment:
The rename of the arguments to `x` and `y` is sufficient to make this point, so no need to explain here. Also, the part about "For spherical/..." is incorrect. That's up to callers to make that convention. There's nothing in the functions that prescribes what each argument means.

Reply:
commit dropped